src/prediction_service_utils.cpp

```diff
@@ -56,126 +57,4 @@ std::map<std::string, shape_t> getRequestShapes(const tensorflow::serving::Predi
     return requestShapes;
 }
 
-Status getModelInstance(ovms::ModelManager& manager,
```

Comment:
Maybe we could get rid of these 2 methods and remove `prediction_service_utils.cpp` completely?

Reply:
It might be a good idea for the future, but in this PR I wouldn't do that. It might be a subject for some wider refactor.